ttl/ttlworker/job_manager.go

```diff
@@ -90,6 +90,7 @@ func NewJobManager(id string, sessPool sessionPool, store kv.Storage) (manager *
 	manager.store = store
 	manager.sessPool = sessPool
 	manager.delCh = make(chan *ttlDeleteTask)
+	manager.notifyStateCh = make(chan interface{})
```

Comment:
maybe we should set the cap of notifyStateCh to 1

Reply:
You're right! :+1: